printer_mods/JosAr/Klicky-Probe/README.md

```diff
@@ -0,0 +1,118 @@
+# Klicky-Probe
+Microswitch probe with magnetic attachement, primarily aimed at CoreXY 3d printers with a focus on the Voron printers, should work on other printers with the variable mount.
+
+The objectives for this project are:
+- drop in replacement for Omron TL-Q5MC2 or PL-08N2 (you don't need to replace the toolhead)
+- easier and faster to build that similiar probe types
+  - does not require soldering
+  - fixed probe dock mount (for the printers that are suported), less variables to adjust
+- be able to detect all the print surfaces
+- be as close to the hotend tip as possible
+- high repetitive and accurate probes
+- less temperature variations
+- no melting of its parts
+- cheap to built
+
+It can also be used with the new ![automatic Z calibration](https://github.com/protoloft/klipper_z_calibration) klipper plugin to efectively calculate the Z offset from the probe and from the Z endstop.
```

Comment:
Remove the `!` (exclamation mark) from the front of that link, exclamation marks are for images, not links.

Replace `efectively` with `effectively`